container_files/assets/www/doc/builtin/sql/ValueExpression.md

```diff
@@ -472,48 +476,60 @@ number of occurrences of those tokens within `str`. For example `tokenize('a b b
 
 ### JSON unpacking <a name="unpack_json"></a>
 
-The `unpack_json(str)` function will parse the string `str` as a JSON object and unpack it into multiple columns following the  algorithm outlined below. Note that JSON objects shown in the tables below are string reprensetations of the JSONs.
+The `unpack_json(str)` function will parse the string `str` as a JSON
```

Comment:
Yes.  I ran out of time to unify the actual function calls, but the internal implementation is unified causing the changes I mentioned.